src/collections/_documentation/platforms/cocoa/index.md

```diff
@@ -90,6 +90,35 @@ _If you crash with a debugger attached, nothing will happen._
 
 Crashes are submitted only upon re-launch of the application. To view the crash in Sentry, close your app and re-launch it.
 
+## Release Health
+
+Monitor the [health of releases]({%- link _documentation/workflow/releases.md -%}#release-health) by observing user adoption, usage of the application, percentage of [crashes]({%- link _documentation/workflow/releases.md -%}#crash), and [session data]({%- link _documentation/workflow/releases.md -%}#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+To benefit from the health data provided in Releases v2, you must use at least version 5.0.0 Beta 2 of the Cocoa SDK and enable the collection of the release health metrics by adding `“enableAutoSessionTracking": true` into options during the initialization of the SDK.
+
+```objective-c
```

Comment:
@bruno-garcia @marandaneto This code block isn't highlighting. Is there a different way Markdown recognizes objective-c? Tried looking through the Markdown docs without much success.

Reply:
[The Internet suggests](https://stackoverflow.com/questions/11890864/is-there-a-github-markdown-language-identifier-for-objective-c-code#comment15825328_11890935) using `smalltalk` to get a better code highlight for objective-c.

Reply:
@bruno-garcia I replaced `objective-c` with `smalltalk` and the highlighting looks really strange. Not sure if that's the best option.

Reply:
which markdown engine are we working with? this might vary, maybe `objc` a try

Reply:
Screenshots pasted here would help